### ISSUE TITLE
node.js path

### ISSUE THREAD

**Issue by kdsingharneja** [csscomb/sublime-csscomb#10]
I am getting the error "node is not recognized as ..."

I can't seem to find where this node's path will go?
I have node.js installed

**Comment by tonyganch** [comment]
You can now set custom path to `node` in settings: https://github.com/csscomb/sublime-csscomb/blob/master/CSScomb.sublime-settings#L4

**Comment by asakasinsky** [comment]
Thank you!

**Comment by CLaBruno** [comment]
I am having the same issue with CSS Comb. I added my Node path in the settings like so:

"node-path" : "C:/Program Files (x86)/nodejs/node",

And I added the path in my system variables as well:

C:\Program Files (x86)\nodejs

When I go to run CSS Comb I am still getting the error:

CSScomb error:
'node' is not recognized as an internal or external command, operable program or batch file.

Any more suggestions?
